packages/design-system-dashboard-cli/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
This is a yarn 3 workspaces thing. We're telling it to use `node-modules` instead of the newer plug-n-play system they have. The other packages in the monorepo have this as well.